### PR TITLE
Pull request/f5716d14

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

File: CUTEst/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,6 @@
+BinDeps
+julia 0.4
```
Comment (tkelman) [inline CUTEst/versions/0.1.0/requires]: If you support Julia 0.4, you should be testing against it on Travis

Comment (abelsiqueira) [inline CUTEst/versions/0.1.0/requires]: Forgot to change it.

After the correction, should I update the tag (0.1.1) or do I force update this?

Comment (tkelman) [inline CUTEst/versions/0.1.0/requires]: Up to you, but if you want to do it as 0.1.1 then we should remove the publishing of 0.1.0 from this branch

**Comment by abelsiqueira** [comment]
Force pushed

**Comment by abelsiqueira** [comment]
This is ready to be reviewed again.

**Comment by tkelman** [comment]
Don't download `https://raw.githubusercontent.com/abelsiqueira/linux-cutest/master/install.sh` from master, that won't be reproducible over time. Either check that script into this repository, or download a specific tag or sha of it.

**Comment by tkelman** [comment]
Using environment variables with spaces in the variable name is a bit unusual.

**Comment by abelsiqueira** [comment]
Updated with the changes here: https://github.com/JuliaSmoothOptimizers/CUTEst.jl/pull/118

**Comment by abelsiqueira** [comment]
Thank you.

**Pull request merged**